imports/api/detectors/methods.js

```diff
@@ -0,0 +1,72 @@
+import { Meteor } from 'meteor/meteor';
+import { ValidatedMethod } from 'meteor/mdg:validated-method';
+import { SimpleSchema } from 'meteor/aldeed:simple-schema';
+import { _ } from 'meteor/underscore';
+
+
+export const matchAffordancesWithDetector = new ValidatedMethod({
+  name: 'detectors.matchAffordancesWithDetector',
+
+  validate: new SimpleSchema({
+    userId: { type: String },
+    detectorId: { type: String }
+  }).validator(),
+
+  run({ userId, detectorId }) {
```

Comment:
this function should get the affordances for the user -- we'll give you their location and a detector and then you will send the location to AA to get the affordances and determine if its a match